rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -385,6 +385,7 @@ def add_cookie(cookie)
         end
 
         def delete_cookie(name)
+          raise ArgumentError, 'Cookie name cannot be null or empty' if name.nil? || name.empty?
```

Comment:
You need a space between the guard class and the method call, and also you can avoid checking for .nil? by changing the if for an unless, unless name || !name.empty?

Reply:
the tests are passing. what needs to be changed now @aguspe ?

Reply:
updated code as per your suggestion @aguspe